<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>备份文件过滤设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名过滤（请使用正则表达式匹配）：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_namereg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>541</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件类型过滤（勾选需要忽略的类型）：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>按时间过滤（忽略在以下时间范围内的文件）：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>访问时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>更改时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>改动时间：</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_ats">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>从</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>从</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>从</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>到</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>到</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>到</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_mts">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_mte">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_cte">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_ate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_cts">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>421</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBox_softlink">
      <property name="text">
       <string>软链接</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_blockdevice">
      <property name="text">
       <string>块设备文件</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_fifo">
      <property name="text">
       <string>管道文件</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_characterdevice">
      <property name="text">
       <string>字符设备文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
